connectors/package.json

```diff
@@ -64,21 +59,25 @@
     "yargs": "^17.7.2"
   },
   "devDependencies": {
-    "@types/eslint": "^8.21.3",
+    "@types/express": "^4.17",
     "@types/fast-levenshtein": "^0.0.2",
+    "@types/fs-extra": "^11.0.1",
+    "@types/minimist": "^1.2.2",
     "@types/morgan": "^1.9.9",
-    "@types/node": "^18.15.5",
+    "@types/node": "^20",
```

Comment:
IMHOP we should at least be stricter on the node version given that our environments are set on `20.13.1`.